content/php/concepts/math-functions/tan/tan.md

```diff
@@ -0,0 +1,30 @@
+#tan()
+
+PHP Calculate Tangent
+
+
+
+PHP’s math functions provides us with functions for calculations involving trigonometry.The tan() function in PHP is used to find the tangent of the argument parameter. The num is in radians and can handle values within the range of integer and float types.
+
+##Syntax
+
+```php
+tan(num); // Returns tan of num
+
+echo(tan(10)); // Returns 0.64836082745
```

Comment:
Remove this and the comments in the syntax block. Also make the syntax block a ` ```pseudo ` code block and include a sentence describing the syntax example after the code block.